systers_portal/meetup/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.conf.urls import url
+
+from meetup.views import MeetupLocationDetailView
+
+
+urlpatterns = [
+    url(r'^(?P<slug>\w+)/(?P<page_slug>\w+)/$',
```

Comment:
That sounds like a strange optimization. The main downside is that for actually inexistent urls (that should give 404), the server will respond with 500 Internal Server Error, because it throws `TemplateDoesNotExist`. 

Could you focus only on main page of the MeetupLocation? One view - one responsibility.